index.js

```diff
@@ -44,7 +44,7 @@ types[VISA] = {
   prefixPattern: /^4$/,
   exactPattern: /^4\d*$/,
   gaps: [4, 8, 12],
```

Comment:
![screen shot 2017-02-06 at 3 16 34 pm](https://cloud.githubusercontent.com/assets/2916945/22666697/47e59ba6-ec7f-11e6-9ee7-25e64319f1c9.png)

Hmm.... If we add the gap, typing 16 digits auto adds the space, which looks like it could be a UX concern (trying to delete the space before submitting).

We're electing to not add gap information until 19 digit cards become more prevalent. 

If anyone feels strongly that we should add the gap information, let us know.